sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -336,23 +336,23 @@ export class StudentService extends RecordDataModelService<Student> {
   /**
    * Updates the student contact information.
    * @param studentId student to be updated.
-   * @param contact contact information to be updated.
+   * @param studentInfo student information to be updated.
    * @param auditUserId user who is making the changes.
    * @returns updated student.
    */
   async updateStudentContactByStudentId(
```

Comment:
The method should be renamed also. Gender is not part of the contact information.